components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -579,7 +579,7 @@ def leave_none_unset_int(s):
           " 'webtest/webclient_plugins/center_plugin.overlay.js.html',"
           " 'channel_overlay_panel']``. "
           "The javascript loads data into ``$('#div_id')``.")],
-    "omero.web.viewer.initial_zoom_level":
+    "omero.client.viewer.initial_zoom_level":
```

Comment:
@dominikl the best if you remove that change from here and I will open another PR once that is merge